cares_reinforcement_learning/memory/memory_buffer.py

```diff
@@ -40,6 +40,29 @@ def sample(self, batch_size):
         transposed_batch = zip(*experience_batch)
         return transposed_batch
 
+    def sample_next(self, batch_size):
+        """
+        For training MBRL to predict rewards. The right next transition is
```

Comment:
as in correct?

Reply:
(s1, a1, r1, s2, done) --> (s2, a2, r2, s3, done)

normal sampling: (s1, a1, r1, s2, done) 
sample_next: (s1, a1, r1, s2, done, a2, r2)